scarb/src/core/manifest/mod.rs

```diff
@@ -67,6 +67,8 @@ pub struct ManifestMetadata {
     #[serde(rename = "tool")]
     pub tool_metadata: Option<BTreeMap<SmolStr, Value>>,
     pub cairo_version: Option<VersionReq>,
+    /// Allow experimental features.
+    pub experimental: Option<ScarbExperimentalFeaturesConfig>,
```

Comment:
```suggestion
    pub experiments: Option<ScarbExperimentalFeaturesConfig>,
```

? noun reads better here than an adjective imo

Reply:
Would `experimental_features` be too long?

Reply:
Kind of? To my understanding, the word `experiments` means basically the same thing, and is more catchy.

On the other hand, if the compiler is going to produce any diagnostics with the term _experimental features_, then it doesn't make sense to change this on our side :/